src/pages/networks/NetworkList.tsx

```diff
@@ -148,25 +149,23 @@ const NetworkList: FC = () => {
               }
             />
           )}
-          {!isLoading && !hasNetworks && (
+          {!isLoading && (
```

Comment:
Why remove the `!hasNetworks` condition?

Reply:
I'm still working on this PR. I'll convert it to Draft. (I'm removing the conditions to force EmptyState render, of course for testing purpose only)